tests/ts_simple_select_is_null.erl

```diff
@@ -0,0 +1,134 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+%% A simple timeseries test to select along the IS [NOT] NULL predicates for
+%% all various datatypes.
+%%
+%% The premise is that if it holds that neither set of IS NULL and IS NOT NULL
+%% satisfying result sets are empty, then the count of rows in the sets combined
+%% matching the count of rows in the set resulting from querying without the
+%% NULL predicates proves that each predicate is correctly implemented.
+
+-module(ts_simple_select_is_null).
+
+-behavior(riak_test).
+
+-export([confirm/0]).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(SPANNING_STEP, (1000)).
+-define(TEST_TYPE, normal).
+
+confirm() ->
+    %% will fail if ts_simple_put_all_null_datatypes fails
+    DDL =
+        "CREATE TABLE GeoCheckin ("
+        " myfamily    varchar     not null,"
+        " myseries    varchar     not null,"
+        " time        timestamp   not null,"
+        " myvarchar   varchar             ,"
+        " myint       sint64              ,"
+        " myfloat     double              ,"
+        " mybool      boolean             ,"
+        " mytimestamp timestamp           ,"
+        " PRIMARY KEY ((myfamily, myseries, quantum(time, 15, 'm')),"
+        " myfamily, myseries, time))",
+    Family = <<"family1">>,
+    Series = <<"seriesX">>,
+    N = 10,
+    Data = make_data(N, Family, Series, []),
```

Comment:
Great call. I'm changing `make_data` to use <<"">> instead of a fuller binary to test this edge since as you said the empty string is the critical edge to test.